first-party/jib-quarkus-extension-maven/README.md

```diff
@@ -63,3 +66,5 @@ Additionally, overriding the following configurations using Maven and Java syste
 
 - `<container><appRoot>`
 - `<container><jvmFlags>`
+
+Set the `fastJarEnabled` property to true if you like to build your image with fast jar.
```

Comment:
We don't update the doc until we make a release. This change should be in a separate PR.